src/fios.h

```diff
@@ -107,6 +107,7 @@ struct FiosItem {
 	uint64 mtime;
 	char title[64];
 	char name[MAX_PATH];
+	bool operator() (const FiosItem& da, const FiosItem& db);
```

Comment:
Wouldn't `operator <` make even more sense here?

Reply:
In this case I'd say yes :)